File: Beginner/FLOW006.py
# cook your dish here
t = int(input())

for _ in range(t):
    
    sum = 0
    rem = 0
    num = int(input())
    tens = 10;
    while(num != 0):
        rem = num%tens
        num = num//tens
        sum += rem
        
    
    print(sum);

File: Beginner/FLOW006.cpp
#include <iostream>
using namespace std;

int main(){
	int t,num,sum,rem,tens;
	
	cin >> t;
	
	for(int i=0;i<t;i++){
		cin >> num;
		sum = 0;
		rem = 0;
		tens = 10;
		while(num !=0){
			rem = num%tens;
			num /= tens;
			sum += rem;
		}
		cout << sum << endl;
	}
	
	return 0;
}